.github/ISSUE_TEMPLATE/product_issue.md

```
@@ -5,6 +5,17 @@ assignees: ''
 
 ---
 
+<!---HEADER START-->
+
+<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)